shaders/src/post_process_inputs.vs

```diff
@@ -3,7 +3,13 @@ LAYOUT_LOCATION(LOCATION_POSITION) in vec4 position;
 LAYOUT_LOCATION(LOCATION_UVS) out vec2 vertex_uv;
 
 struct PostProcessVertexInputs {
+
+    // We provide normalized (uv) and non-normalized (texelCoords) texture coordinates to
+    // custom vertex shaders. By default the non-normalized coordinates are passed through
```

Comment:
I'm wondering if we should rename the `getUV()` function in `post_process_inputs.fs` to something like `getTexelCoordinates()` to avoid confusion.

Reply:
I think we should have both.